dev/com.ibm.ws.microprofile.faulttolerance.2.0/bnd.bnd

```diff
@@ -0,0 +1,82 @@
+#*******************************************************************************
+# Copyright (c) 2017,2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: com.ibm.ws.microprofile.faulttolerance.2.0
+Bundle-SymbolicName: com.ibm.ws.microprofile.faulttolerance.2.0
+
+Bundle-Description:MicroProfile Fault Tolerance 2.0 impl, version ${bVersion}
+
+WS-TraceGroup: FAULTTOLERANCE
+
+-dsannotations-inherit: true
+-dsannotations: com.ibm.ws.microprofile.faulttolerance20.impl.ProviderResolverImpl20, \
+                com.ibm.ws.microprofile.faulttolerance.metrics.impl.DummyRecorderProvider
+
+Import-Package: \
+  org.eclipse.microprofile.faulttolerance; version="[1.0,1.3)",\
+  *
+
+Export-Package: 
+
+Private-Package: \
+        com.ibm.ws.microprofile.faulttolerance.resources,\
+        com.ibm.ws.microprofile.faulttolerance.impl, \
+        com.ibm.ws.microprofile.faulttolerance.impl.policy, \
+        com.ibm.ws.microprofile.faulttolerance.impl.sync, \
+        com.ibm.ws.microprofile.faulttolerance.impl.async, \
+        com.ibm.ws.microprofile.faulttolerance.metrics.impl, \
+        com.ibm.ws.microprofile.faulttolerance20.*
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+Include-Resource: \
+  @${repo;com.ibm.ws.microprofile.faulttolerance}!/!(OSGI-INF)/*
```

Comment:
Open an issue to refactor out a common bundle

Reply:
#5529